Formula/r/rapidjson.rb

```diff
@@ -22,10 +22,12 @@ class Rapidjson < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "doxygen" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args
+    system "cmake", "-DRAPIDJSON_BUILD_DOC=OFF",
```

Comment:
why turning off the docs?

Reply:
why are they required? 
People can read online docs http://rapidjson.org instead of having them in the package